### ISSUE TITLE
Missing instructions - not much "teaching"

### ISSUE THREAD

**Issue by jsmwrench** [dotnet/AspNetCore.Docs#26317]
This tutorial worked as intended, but for those who are actual "beginners", I feel this tutorial will be hard to follow.  There really isn't any "teaching" in this tutorial.  It's mainly a tutorial on following instructions and copy and pasting.  There is very little description describing the code that is copied and pasted or any explanation as to how the code works.

Also, the code in the section "Add SignalR client code" there is an instruction that says "In the wwwroot/js folder, create a chat.js file with the following code:" but no instruction on HOW to make this "chat.js" file.  I figured it out, but it wasn't intuitive.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0dd2991d-c22d-ad0f-80b6-bd1b503db8f3
* Version Independent ID: 5e3fbe1d-c97f-9940-4f0c-067d4b2112b1
* Content: [Get started with ASP.NET Core SignalR](https://docs.microsoft.com/en-us/aspnet/core/tutorials/signalr?view=aspnetcore-6.0&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/signalr.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/signalr.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @bradygaster
* Microsoft Alias: **bradyg**

**Comment by guardrex** [comment]
@jsmwrench ... That's mostly by-design. The tutorials are intended to get devs up and running with a quick, basic example *without a lot of explanation and cross-links*. The reference docs cover the details. In this case WRT your remark about the JS file, you'd learn more about that in ...

https://docs.microsoft.com/aspnet/core/signalr/javascript-client
